src/components/FeaturedProjects.js

```diff
@@ -0,0 +1,96 @@
+import React, { useState } from 'react'
+import styled from 'styled-components'
+import { OuterWrapper, InnerWrapper, Heading, Topics, ProjectsHeading, TopicsContainer } from './Styling'
+
+const GithubAPI = 'https://api.github.com/users/marwebdesign/repos'
+
+export const FeaturedProjects = () => {
+  const [featuredProjects, setFeaturedProjects] = useState([''])
+  fetch(GithubAPI)
+    .then((res) => res.json())
+    .then((data) => {
+      setFeaturedProjects(data)
+    })
+
+  const BigProjects = featuredProjects.filter((firstProjects) => (firstProjects.name === 'project-movies') || (firstProjects.name === 'project-survey') || (firstProjects.name === 'project-chatbot') || (firstProjects.name === 'project-guess-who'));
+  return (
+    <OuterWrapper beige>
+      <Heading dark>FEATURED PROJECTS</Heading>
+      <InnerWrapper>
+        <FeaturedProjectsContent>
+          {BigProjects.map((prop) => (
+            <FeaturedProjectsContainer key={prop.id}>
+              <ProjectLink href={prop.html_url} alt="Link to project" target="_blank">
+                <ImageAndName>
+                  <BigProjectsImage alt="project-img" src={`https://raw.githubusercontent.com/marwebdesign/${prop.name}/master/code/Thumbnail/thumbnail.png`} />
+                  <ProjectNameOverImage>{prop.name}</ProjectNameOverImage>
+                </ImageAndName>
+              </ProjectLink>
+              <ProjectsHeading>{prop.name}</ProjectsHeading>
+              <p>{prop.description}</p>
+              <TopicsContainer>
+                {prop.topics.map((tag) => (
+                  <Topics>{tag}</Topics>
```

Comment:
Would it be possible to move down the closing tag od ProjectLink to wrap the whole "section" in with the link?